### PR TITLE
fix: make the file correct

### PR DESCRIPTION
**Pull request by vcokltfre**
I was too lazy to MITM the requests, so I made a pr instead

### CODE REVIEW

**Review by ChrisLovering — APPROVED** [review]
LGTM

**Review by Akarys42 — APPROVED** [review]
That's the quality PR I expected from the UK quality standards

**Review by Bluenix2 — APPROVED** [review]
It is good that the rest of the world gets to take part in some highly trustable UK enginerring among Brexit. Thank you

File: site/index.html [hunk 1]
```diff
@@ -5,9 +5,9 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
         <link rel="stylesheet" href="static/style.css" />
-        <title>No Wungus</title>
+        <title>Yes Wungus</title>
```
Comment (fisher60) [inline site/index.html:8]:
```suggestion
        <title>Absolutely No Wungus</title>
```

File: site/index.html [hunk 2]
```diff
@@ -5,9 +5,9 @@
...
     </head>
     <body>
-        <h1>No</h1>
+        <h1>Wungus!</h1>
```
Comment (fisher60) [inline site/index.html:11]:
```suggestion
        <h1>No Wungus!</h1>
```

**Review by fisher60 — CHANGES_REQUESTED** [review]
Quality of code is far above expected, so very good job. However there are a few things I would like to see changed. I think these changes better capture the spirit of this project and promote the health of multiple communities. Make these changes and we can revisit merging this PR. Thank you.

**Comment by fisher60** [comment]
ye no

**Pull request closed without merging**